src/handlers/on_comment.py

```diff
@@ -132,15 +127,15 @@ def rollback_file(repo_full_name, pr_path, installation_id, pr_number):
 
     # Get the file's content from the previous commit
     commits = repo.get_commits(sha=branch_name)
-    if commits.totalCount < 2:
+    if commits.totalCount < 3:
```

Comment:
this isn't the way to do it, we should keep the existing paths, and add a path on line 140 which checks the commit messages of the last two paths. Ex: They should match the "Revert file ..." string